src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,10 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public int compareTo(Person otherPerson){
```

Comment:
Comment for this method would be helpful. What property/criteria does this method use to compare?

Reply:
Noted with thanks.